Sources/Sentry/SentrySerialization.m

```diff
@@ -100,32 +101,33 @@ + (SentryEnvelope *_Nullable)envelopeWithData:(NSData *)data
                                                                                error:&error];
             if (nil != error) {
                 SENTRY_LOG_ERROR(@"Failed to parse envelope header %@", error);
-            } else {
-                SentryId *eventId = nil;
-                NSString *eventIdAsString = headerDictionary[@"event_id"];
-                if (nil != eventIdAsString) {
-                    eventId = [[SentryId alloc] initWithUUIDString:eventIdAsString];
-                }
-
-                SentrySdkInfo *sdkInfo = nil;
-                if (nil != headerDictionary[@"sdk"]) {
-                    sdkInfo = [[SentrySdkInfo alloc] initWithDict:headerDictionary];
-                }
-
-                SentryTraceContext *traceContext = nil;
-                if (nil != headerDictionary[@"trace"]) {
-                    traceContext =
-                        [[SentryTraceContext alloc] initWithDict:headerDictionary[@"trace"]];
-                }
-
-                envelopeHeader = [[SentryEnvelopeHeader alloc] initWithId:eventId
-                                                                  sdkInfo:sdkInfo
-                                                             traceContext:traceContext];
-
-                if (headerDictionary[@"sent_at"] != nil) {
-                    envelopeHeader.sentAt = sentry_fromIso8601String(headerDictionary[@"sent_at"]);
-                }
+                break;
+            }
```

Comment:
I added the break here to reduce the code indentation. No changes to the code itself.